api/users.rb

```diff
@@ -183,4 +187,16 @@ def set_template_result(user_id, user_stats, thread_ids)
   end
 end
 
+get "#{APIPREFIX}/users/:user_id/social_stats" do |user_id|
+  begin
+    _user_social_stats(user_id, params)
+  end
+end
+
+post "#{APIPREFIX}/users/:user_id/social_stats" do |user_id|
```

Comment:
@mtyaka because `GET` is limited to 4096 symbols if I'm not mistaking, and theoretically, we can have a lot of cohorted discussions. So I decided it is now best to use post to query for such data.